documentation/correctness/overview.adoc

```diff
@@ -0,0 +1,46 @@
+= Serialization chunk correctness
+
+Whether a model is _correct_ —or _valid_, or _consistent_— is an important question.
+
+[IMPORTANT]
+====
+We'll talk about _correctness_ from now, and consider _validity_ and _consistency_ to be equivalent concepts.
+====
+
+Correctness ultimately determines whether a model is useful in the context it exists and is used in.
+Answering that question only with a simple "`yes`" or "`no`" is too simplistic: a model can be incorrect in different ways, and on different levels.
+Having an illegal trailing comma in an array somewhere in the JSON serialization of a model —e.g., `[ 0, 1, 2**,** ]`— should not prevent that serialization chunk from being consumable — i.e.: processable, interchangeable, deserializable, etc.
```

Comment:
if we really want to mention it, I'd move the `i.e.: processable, interchangeable, deserializable, etc.` part to a footnote: This sentence is already hard enough to read without it.